src/App.tsx

```diff
@@ -93,6 +94,10 @@ function App() {
             path="/events/:eventId/ccps/:ccpId/patients/:patientId"
             component={(props) => <PatientProfilePage mode="edit" {...props} />}
           />
+          <Route
+            path="/events/:eventId/ccps/:ccpId/patientDetails/:patientId"
```

Comment:
@petorsong I'm assuming this is the route that was decided upon?

Reply:
(just commenting here to address the PR comment as well)

i like the `.../view` route and the idea of not introducing a "duplicate" component if we can reuse our existing one